### PR TITLE
Thorium DLL Added, Starter Bag Changes, and Some Shimmer Items Locked Behind Progression

### PR DESCRIPTION
**Pull request by DaChickenGamer**
Thorium got approved it seems!

### CODE REVIEW

**Comment by DaChickenGamer** [comment]
Here is the proof to it being approved:

https://discordapp.com/channels/919631817944993802/1127624443032780891/1309255792473407508

**Comment by 1Git2Clone** [comment]
Awesome! Unfortunately I'll have to be off the PC for 2-3 days so I can't check the integrity of the DLL in order to merge it until next week

**Comment by DaChickenGamer** [comment]
> Awesome! Unfortunately I'll have to be off the PC for 2-3 days so I can't check the integrity of the DLL in order to merge it until next week

All good take as much time as you need.

File: CONTRIBUTING.md
Comment (1Git2Clone) [inline CONTRIBUTING.md:1]: The markdown file is practically the same, this change is redundant.

File: Content/BootRecipes.cs
Comment (1Git2Clone) [inline Content/BootRecipes.cs:1]: Unneeded. This also causes a merge conflict so make sure to resolve it.

File: Common/Systems/BossRushInjectionSystem.cs
Comment (1Git2Clone) [inline Common/Systems/BossRushInjectionSystem.cs:1]: Make sure to resolve this merge conflict too, it should automatically be resolved just by doing `git merge main`.

Comment (1Git2Clone) [reply Common/Systems/BossRushInjectionSystem.cs:1]: ^ Other than that it looks good to me.

File: Content/Items/Accessories/FrigidEnchantment.cs
Comment (1Git2Clone) [inline Content/Items/Accessories/FrigidEnchantment.cs:1]: Why are unrelated refactors like these in the PR? In any case, looks good to me.

File: README.md
Comment (1Git2Clone) [inline README.md:1]: What linter did you use to get this one? Revert this part.

File: lib/ThoriumMod_v1.7.1.6.dll
Comment (1Git2Clone) [inline lib/ThoriumMod_v1.7.1.6.dll:1]: Valid DLL (Looks good to me).

```sh
$ sha256sum $(pwd)/ThoriumMod.dll ~/Downloads/ThoriumMod_v1.7.1.6.dll
236129bd5ce0e04a12fc97d540e05ddb2ddd6725be55bdef2e9a096e10df2e71  /home/hutao/.local/share/Terraria/tModLoader/ModReader/ThoriumMod/ThoriumMod.dll
236129bd5ce0e04a12fc97d540e05ddb2ddd6725be55bdef2e9a096e10df2e71  /home/hutao/Downloads/ThoriumMod_v1.7.1.6.dll
```

**Review by 1Git2Clone — CHANGES_REQUESTED** [review]
1. Make sure to resolve the merge conflicts (check #45 and #42 to see what the conflicts were about).
2. Make sure to revert the `CONTRIBUTING.md` and `README.md` changes. This project uses markdownlint<sup>\[[1]\]</sup> if you're questioning the formatting of these files.

---

Only after these two things are done I can test out the changes in: `Content/RecipeChanges/ShimmerRecipeChanges.cs` and `Content/ItemChanges/CalamityStarterBagChanges.cs`.

[1]: https://github.com/markdownlint/markdownlint/

File: Content/Items/Accessories/FrigidEnchantment.cs
Comment (DaChickenGamer) [reply Content/Items/Accessories/FrigidEnchantment.cs:1]: Yeah most of these were caused by running the clean code function in Jetbrains Rider.

**Comment by 1Git2Clone** [comment]
Looks good to me!

**Review by 1Git2Clone — APPROVED** [review]

**Pull request merged**